### DISCUSSION TITLE
bedanken für die arbeit und deinen zeitaufwand

### DISCUSSION THREAD

**Discussion by DerMicha23** [Mogli12/KeyboardSteer#731] [General]
Nur mal einfach danke sagen denn das machen zu wenige.
also DANKE

**Comment by Mogli12** [comment]
Vielen Dank! Ich werde jetzt einfach versuchen, an diesem Mod hier auf GitHub ohne Stress weiterzumachen.

**Comment by fdanapfel** [comment]
Auch von mir einen ganz großen Dank für all die Zeit und Arbeit die du in diesen Mod investierst!

Ich möchte auf diesen Mod im LS nicht mehr verzichten!

**Comment by Williwillswisse** [comment]
Auch von mir ein großes Dankeschön. Ich werde deine Mods im 19er vermissen.

**Comment by Laurindra** [comment]
A big thanks from me as well. I've really appreciated the  details and realism in FS17 Gearbox Addon and I currently miss it in FS19 very much. I hope you consider continueing writing this for FS19 and I will follow it here on GitHub.

**Comment by Shreklich** [comment]
Hallo möchte mich auch gerne für deinen Mod bedanken.

**Comment by Mogli12** [comment]
If you are looking for a more realistic transmission please visit https://github.com/modelleicher/FS19_realManualTransmission

**Comment by Moselbruecke** [comment]
Vielen Dank für deine Arbeit. 
Besonders freut mich, das GPS um 90Grad wechseld, so das man bequem ums Feld fahren kann.
Ich hab noch einen Wunsch zum Getriebe:
4, 5 und 6 Gänge für Oldtimer und Mähdrescher, da diese nicht so viele Gänge haben wie moderne Fahrzeuge.

**Comment by Toidi2k** [comment]
Super Arbeit nutzen nun alle nur noch deinen Mod auf den 2 Servern wo ich spiele. Alle sind begeistert von dem einfach zu nutzenden GPS und auch dem Getriebe. Weiter so.

**Comment by Sacred-Shadow** [comment]
So ein PayPal Donation Link wär schon cool ... weiß nur nich wie Giants das findet.

**Comment by rondo24xx** [comment]
Auch von mir ein großes Dankeschön für Deine Arbeit.
Ist mittlerweile schon so fester Bestandteil in meinem Gameplay, das man es
fast schon zum Basisgame zählt.

**Comment by Hmmvee** [comment]
Da will ich mich auch direkt anhängen.
Herzlichen Dank für den Mod, für mich als Tastaturfahrer ein Traumzusatz der so eigentlich im Spiel sein sollte.
